src/Robo/Commands/Tests/DrupalTestCommand.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Acquia\Blt\Robo\Commands\Tests;
+
+use Acquia\Blt\Robo\Exceptions\BltException;
+use Robo\Contract\VerbosityThresholdInterface;
+
+/**
+ * Defines commands in the "tests" namespace.
+ */
+class DrupalTestCommand extends TestsCommandBase {
+
+  const APACHE_RUN_GROUP = 'APACHE_RUN_GROUP';
```

Comment:
Why are these set up as constants rather than var? Just want to understand the thinking.